BubbleSort/bubble_sort.rb

```diff
@@ -24,6 +24,28 @@ def bucle(input)
   input
 end
 
+def bubble_sort(arr)
+  runs = arr.size -1
+  return arr unless runs < 0
+
+  while changes > 1
+    changes = 0
+
+    0.upto(runs) do |i|
+      arr[i], arr[i + 1] = arr[i + 1], arr[i] if arr[i] > arr[i + 1]
+    end
+    runs -= 1
+  end
+  arr
+end
+end
+end
+
 # testing
+<<<<<<< HEAD
 puts bubble_sort([4, 3, 78, 2, 0, 2]) # [0,2,2,3,4,78]
 puts bubble_sort([3, 2]) # [2,3]
+=======
+bubble_sort([4, 3, 78, 2, 0, 2]) # [0,2,2,3,4,78]
+bubble_sort([3, 2]) # [2,3]
+>>>>>>> ba2cd529699876f97b50bf508a71565b345fe1db
```

Comment:
Those `<<<< HEAD` symbols mean there was a merge conflict. This file won't work since it has all that gibberish. Please fix